.github/workflows/test-coverage-enhancement.yml

```
@@ -52,6 +53,7 @@
             testing
             automated-pr
       - name: Trigger CodeRabbit
+        if: ${{ success() && github.event_name != 'pull_request' }}  
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)